src/docs/product/data-management-settings/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,130 @@
+
+When modifying data in the file, Sentry cannot modify the length of the file. This has implications for how the [Methods](../advanced-datascrubbing/#methods) behave:
+
+- Any modification that results in replacement text **longer** than the original will be **truncated**.
+- Any modification that results in replacement text **shorter** than the original will be **padded** using `x` as the padding character.
+
+Here is how this works specifically for each method:
+
+- *Remove*: As the entire field is removed, it is essentially replaced with the padding character.
+- *Mask*: This behaves as normal, all data is replaced with `*` masking character.
+- *Hash*: Hashing happens as usual, however the **binary representation** of the match is hashed, regardless of the encoding in which the match was made.  The resulting hash is padded or truncated as described above.
+- *Replace*: This behaves like normal, with the replacement being padded or truncated as described above.
+
+
+## Attachment Source Selectors
+
+All attachments can be selected using the `$attachments` [Value Type](../advanced-datascrubbing/#value-types) selector at the root of the selector, followed by the specific filename.
+The filename must be in quotes to write a specific filename, but wildcards can also be used.
+
+For example, for the filename `minidump.dmp`, you can select all fields to be scrubbed using `$attachments.'minidump.dmp'.**`.
+To write the same using a wildcard to select all attachments, you would write `$attachments.*.**` or even simply `$attachments.**`.
+
+
+### Minidump Selectors
+
+A minidump is always an attachment, but it also has its own [value type](../advanced-datascrubbing/#value-types) of `$minidump`, which allows you to select all fields that can be scrubbed in a minidump using `$minidump.**`.
+This really is a shorthand for selecting the fields of all attachments that are minidumps: `$attachments.$minidump.**`.
+
+Our examples thus far have repeatedly referred to all fields in a minidump that can be scrubbed.
+For the purpose of PII scrubbing, we currently represent the minidump as a flat structure with the following fields:
+
+- The **stack memory**: These are memory regions in the minidump that are used as stack memory by a thread.
+  You can use the `stack_memory` path item to select these regions.
+  Generally they will contain binary data, so be careful with the [data types](../advanced-datascrubbing/#data-types) used to scrub these regions.
+
+  We **do not recommend** scrubbing stack memory since it is required by Sentry to build a readable stack trace for the event.
+  Inadvertently modifying the stack in an undesired way will make this process impossible and the event far less useful.
+  Because of this there is no way to select the stack memory by a more generic selector like a [value type](../advanced-datascrubbing/#value-types), you must select it explicitly e.g. using `stack_memory` or `$minidump.stack_memory` for it to be scrubbed.
```

Comment:
suggest:
Because of this, you cannot select the stack memory by a more generic selector like a [value type](../advanced-datascrubbing/#value-types). You must select it explicitly, such as by `stack_memory` or `$minidump.stack_memory` for it to be scrubbed.